CHANGELOG.asciidoc

```diff
@@ -64,6 +64,7 @@ This release also includes changes from <<release-3-6-7, 3.6.7>>.
 * TINKERPOP-3021 Publish ARM64 Gremlin Console Images
 * TINKERPOP-3030 Update to .NET 8
 * TINKERPOP-3068 Make serviceName and mergedParams public for provider usage in CallStep
+* TINKERPOP-3082 Allow specifying a customized Spark app name
```

Comment:
Sorry, I should have been more specific. Please remove this and add something like `Allow specifying a customized Spark app name` to https://github.com/apache/tinkerpop/blob/3.7-dev/CHANGELOG.asciidoc?plain=1#L30

This "Improvements" section is automatically generated during release so it shouldn't be manually modified.